resources/assets/js/videos/components/videoScreen.vue

```diff
@@ -540,6 +571,24 @@ export default {
         heightOffset() {
             this.updateSize();
         },
+        jumpByFrameEnabled(enabled) {
+            if(enabled) {
+                Keyboard.off('ArrowRight', this.emitNext, 0, this.listenerSet);
+                Keyboard.off('ArrowLeft', this.emitPrevious, 0, this.listenerSet);
+                Keyboard.on('Shift+ArrowRight', this.emitNext, 0, this.listenerSet);
+                Keyboard.on('Shift+ArrowLeft', this.emitPrevious, 0, this.listenerSet);
+                Keyboard.on('ArrowRight', this.emitNextFrame, 0, this.listenerSet);
+                Keyboard.on('ArrowLeft', this.emitPreviousFrame, 0, this.listenerSet);
+            }
+            else {
+                Keyboard.off('Shift+ArrowRight', this.emitNext, 0, this.listenerSet);
+                Keyboard.off('Shift+ArrowLeft', this.emitPrevious, 0, this.listenerSet);
+                Keyboard.off('ArrowRight', this.emitNextFrame, 0, this.listenerSet);
+                Keyboard.off('ArrowLeft', this.emitPreviousFrame, 0, this.listenerSet);
+                Keyboard.on('ArrowRight', this.emitNext, 0, this.listenerSet);
+                Keyboard.on('ArrowLeft', this.emitPrevious, 0, this.listenerSet);                
+            }
+        },
```

Comment:
This only works because `jumpByFrameEnabled` is `false` initially and set to `true` later. If it was `true` initially, the keyboard shortcuts to jump by frames would not be enabled. I suggest that you move this to a separate method. Then you call the method both in the `created()` hook (delete the arrow shortcuts there) and in the `jumpByFrameEnabled` watcher.